examples/DataApi-Voting-poll/template/modules/shared_data/src/main/scala/com/my/currency/shared_data/Main.scala

```diff
@@ -0,0 +1,86 @@
+package com.my.currency.shared_data
+
+import cats.conversions.all.autoWidenFunctor
+import cats.data.NonEmptyList
+import cats.effect.IO
+import cats.implicits.catsSyntaxValidatedIdBinCompat0
+import com.my.currency.shared_data.Utils.getLastMetagraphIncrementalSnapshotInfo
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+import org.tessellation.currency.dataApplication.dataApplication.DataApplicationValidationErrorOr
+import org.tessellation.currency.dataApplication.{DataApplicationValidationError, DataState, L0NodeContext, L1NodeContext}
+import org.tessellation.security.signature.Signed
+import com.my.currency.shared_data.combiners.Combiners.{combineCreatePoll, combineVoteInPoll}
+import com.my.currency.shared_data.errors.Errors.CouldNotGetLatestCurrencySnapshot
+import com.my.currency.shared_data.validations.Validations.{createPollValidations, createPollValidationsWithSignature, voteInPollValidations, voteInPollValidationsWithSignature}
+import com.my.currency.shared_data.types.Types.{CreatePoll, PollUpdate, VoteCalculatedState, VoteInPoll, VoteStateOnChain}
+import org.slf4j.LoggerFactory
+import org.tessellation.security.SecurityProvider
+
+object Main {
```

Comment:
If this file is not meant to be executed then maybe it shouldn't be called `Main`